pay-api/src/pay_api/services/statement.py

```diff
@@ -163,3 +168,26 @@ def get_statement_report(statement_id: str, content_type: str, template_name='st
         current_app.logger.debug('>get_statement_report')
 
         return report_response, report_name
+
+    @staticmethod
+    def get_summary(auth_account_id: str):
+        """Get summary for statements by account id."""
+        # This is written outside of the model, because we have multiple model references that need to be included.
+        # If we include these references inside of a model, it runs the risk of having circular dependencies.
+        # It's easier to build out features if our models don't rely on other models.
```

Comment:
Ideally resources rely on services should rely on models, models should never rely on models - Otherwise you're in for a world of pain of circular references.